scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -148,16 +148,9 @@ def test_vertices_regions_scaling_invariance(self):
         assert_array_almost_equal(sv_unit.vertices * 2,
                                   sv_scaled.vertices)
 
-    def test_old_radius_api(self):
-        sv_unit = SphericalVoronoi(self.points, radius=1)
-        with suppress_warnings() as sup:
-            sup.filter(DeprecationWarning, "`radius` is `None`")
-            sv = SphericalVoronoi(self.points, None)
-            assert_array_almost_equal(sv_unit.vertices, sv.vertices)
-
     def test_old_radius_api_warning(self):
-        with assert_warns(DeprecationWarning):
-            SphericalVoronoi(self.points, None)
+        with pytest.raises(ValueError, match='`radius` is `None`. *'):
+            SphericalVoronoi(self.points, radius=None)
```

Comment:
Should we rename this test to something like `test_old_radius_api_error` since it doesn't emit a warning anymore?

Reply:
Yes, you are right, good idea.

Reply:
How do I fix the error with the `numpydoc` submodule? I never opened the file `doc/sphinxext`, so I'm not sure how I made that change.

Reply:
Perhaps try rebasing on/merging with `main`? The `doc/sphinxext` path is no longer listed in `.gitmodules` on main.

Reply:
Thanks a lot @rossbar.